pom.xml

```diff
@@ -62,23 +63,35 @@
     <beeju.version>1.2.1</beeju.version>
     <hamcrest-library.version>1.3</hamcrest-library.version>
     <housekeeping.version>1.0.2</housekeeping.version>
+    <guava.version>20.0</guava.version>
+    <google-cloud.version>0.34.0-alpha</google-cloud.version>
```

Comment:
This and the next one are only used in gcp-ct right? If so we should move them down there.